src/main/java/io/xstefank/wildfly/bot/util/PullRequestLogger.java

```diff
@@ -0,0 +1,116 @@
+package io.xstefank.wildfly.bot.util;
+
+import org.jboss.logging.Logger;
+import org.kohsuke.github.GHPullRequest;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class PullRequestLogger {
+    private static final Map<String, Logger> delegates = new HashMap<>();
```

Comment:
I don't understand why do you need multiple delegates

Reply:
Sure, I can change it only to the single field

Reply:
still map?

Reply:
So we do not have to recreate Logger on every pullRequest, i.e. our beans are request scoped. If we receive 3 PRs only the first one will create jboss Loggers and cache them in this map. Next 2 pull request will not recreate them.